prov/efa/src/rdm/rxr_pkt_type_req.c

```diff
@@ -1799,33 +1799,33 @@ void rxr_pkt_handle_longcts_rtw_sent(struct rxr_ep *ep,
 void rxr_pkt_handle_eager_rtw_send_completion(struct rxr_ep *ep,
 					      struct rxr_pkt_entry *pkt_entry)
 {
-	struct rxr_op_entry *tx_entry;
+	struct efa_rdm_ope *tx_entry;
 
-	tx_entry = (struct rxr_op_entry *)pkt_entry->x_entry;
+	tx_entry = (struct efa_rdm_ope *)pkt_entry->x_entry;
 	assert(tx_entry->total_len == rxr_pkt_req_data_size(pkt_entry));
-	rxr_op_entry_handle_send_completed(tx_entry);
+	efa_rdm_ope_handle_send_completed(tx_entry);
 }
 
 void rxr_pkt_handle_longcts_rtw_send_completion(struct rxr_ep *ep,
 					     struct rxr_pkt_entry *pkt_entry)
 {
-	struct rxr_op_entry *tx_entry;
+	struct efa_rdm_ope *tx_entry;
 
-	tx_entry = (struct rxr_op_entry *)pkt_entry->x_entry;
+	tx_entry = (struct efa_rdm_ope *)pkt_entry->x_entry;
 	tx_entry->bytes_acked += rxr_pkt_req_data_size(pkt_entry);
 	if (tx_entry->total_len == tx_entry->bytes_acked)
-		rxr_op_entry_handle_send_completed(tx_entry);
+		efa_rdm_ope_handle_send_completed(tx_entry);
 }
 
 /*
  *     handle_recv() functions
  */
 
 static
-struct rxr_op_entry *rxr_pkt_alloc_rtw_rx_entry(struct rxr_ep *ep,
+struct efa_rdm_ope *rxr_pkt_alloc_rtw_rx_entry(struct rxr_ep *ep,
 						struct rxr_pkt_entry *pkt_entry)
```

Comment:
alignment... sigh.... are these worth tracking down?

Reply:
I would like to handle them via one patch when we are done with all the renaming.